antora/docs/modules/ROOT/pages/release_policy.adoc

```diff
@@ -0,0 +1,1707 @@
+
+| [#slsa3]`slsa3`
+a| Includes policy rules required to meet SLSA Level 3.
+
+Rules included:
+
+* xref:release_policy.adoc#slsa_build_build_service__allowed_builder_ids_provided[Allowed builder IDs provided]
+* xref:release_policy.adoc#slsa_provenance_available__allowed_predicate_types_provided[Allowed predicate types provided]
+* xref:release_policy.adoc#slsa_build_scripted_build__build_script_used[Build task contains steps]
+* xref:release_policy.adoc#slsa_build_scripted_build__build_task_image_results_found[Build task set image digest and url task results]
+* xref:release_policy.adoc#slsa_provenance_available__attestation_predicate_type_accepted[Expected attestation predicate type found]
+* xref:release_policy.adoc#slsa_source_correlated__expected_source_code_reference[Expected source code reference]
+* xref:release_policy.adoc#slsa_source_version_controlled__materials_uri_is_git_repo[Material uri is a git repo]
+* xref:release_policy.adoc#slsa_source_version_controlled__materials_format_okay[Materials have uri and digest]
+* xref:release_policy.adoc#slsa_source_version_controlled__materials_include_git_sha[Materials include git commit shas]
+* xref:release_policy.adoc#tasks__pipeline_has_tasks[Pipeline run includes at least one task]
+* xref:release_policy.adoc#slsa_build_scripted_build__subject_build_task_matches[Provenance subject matches build task image result]
+* xref:release_policy.adoc#slsa_source_correlated__rule_data_provided[Rule data provided]
+* xref:release_policy.adoc#slsa_build_build_service__slsa_builder_id_found[SLSA Builder ID found]
+* xref:release_policy.adoc#slsa_build_build_service__slsa_builder_id_accepted[SLSA Builder ID is known and accepted]
+* xref:release_policy.adoc#slsa_source_correlated__source_code_reference_provided[Source code reference provided]
+* xref:release_policy.adoc#slsa_source_correlated__attested_source_code_reference[Source reference]
+* xref:release_policy.adoc#tasks__successful_pipeline_tasks[Successful pipeline tasks]
+|===
+
+[#attestation_type_package]
+== link:#attestation_type_package[Attestation type]
+
+Sanity checks related to the format of the image build's attestation.
+
+* Package name: `attestation_type`
+* Package full path: `policy.release.attestation_type`
+
+[#attestation_type__deprecated_policy_attestation_format]
+=== link:#attestation_type__deprecated_policy_attestation_format[Deprecated policy attestation format]
+
+The Enterprise Contract CLI now places the attestation data in a different location. This check fails if the expected new format is not found.
+
+*Solution*: Use a newer version of the Enterprise Contract CLI.
+
+* Rule type: [rule-type-indicator failure]#FAILURE#
+* FAILURE message: `Deprecated policy attestation format found`
+* Code: `attestation_type.deprecated_policy_attestation_format`
+* Effective from: `2023-08-31T00:00:00Z`
+* https://github.com/enterprise-contract/ec-policies/blob/main/policy/release/attestation_type.rego#L72[Source, window="_blank"]
```

Comment:
I see what you mean about this easily getting out of date. It's not tied to a particular commit ID so even when things are working properly, there's a period of time where this will inevitably point to the wrong place. I think that's ok. This is a nice-to-have feature IMO. We can try to come up with something clever in the future if needed.

Reply:
Tweaked it a bit, might be okay now